docs/configure/general.asciidoc

```diff
@@ -6,210 +6,208 @@ General APM Server configuration options.
-+
+
 |====
 | APM Server binary | N/A
 | Fleet-managed     | `URL`
 |====
 
-Maximum request header::
+[[max_header_size]]
+[float]
+== Max header size
 Maximum permitted size of a request's header accepted by the server to be processed (in Bytes).
-+
+Defaults to 1048576 Bytes (1 MB). (int)
+
 |====
 | APM Server binary | `apm-server.max_header_size`
 | Fleet-managed     | `Maximum size of a request's header`
 |====
 
-
-
-
-
-
-Example config file:
-
-["source","yaml"]
-----
-apm-server:
-  host: "localhost:8200"
-  rum:
-    enabled: true
-
-output:
-  elasticsearch:
-    hosts: ElasticsearchAddress:9200
-
-max_procs: 4
-----
-
-NOTE: If you are using an X-Pack secured version of {stack},
-you need to specify credentials in the config file before you run the commands that set up and start APM Server.
-For example:
-
-[source,yaml]
-----
-output.elasticsearch:
-  hosts: ["ElasticsearchAddress:9200"]
-  username: "elastic"
-  password: "elastic"
-----
-
-[float]
-[[configuration-apm-server]]
-== Configuration options: `apm-server.*`
-
-[[host]]
-[float]
-=== `host`
-Defines the host and port the server is listening on.
-Use `"unix:/path/to.sock"` to listen on a Unix domain socket.
-Defaults to 'localhost:8200'.
-
-[[max_header_size]]
-[float]
-=== `max_header_size`
-Maximum permitted size of a request's header accepted by the server to be processed (in Bytes).
-Defaults to 1048576 Bytes (1 MB).
-
 [[idle_timeout]]
 [float]
-=== `idle_timeout`
+== Idle timeout
 Maximum amount of time to wait for the next incoming request before underlying connection is closed.
-Defaults to 45 seconds.
+Defaults to `45s` (45 seconds). (text)
+
+|====
+| APM Server binary | `apm-server.idle_timeout`
+| Fleet-managed     | Idle time before underlying connection is closed
+|====
 
 [[read_timeout]]
 [float]
-=== `read_timeout`
+== Read timeout
 Maximum permitted duration for reading an entire request.
-Defaults to 30 seconds.
+Defaults to `3600s` (3600 seconds). (text)
+
+|====
+| APM Server binary | `apm-server.read_timeout`
+| Fleet-managed     | Maximum duration for reading an entire request
+|====
+
+// REVIEWER: Default 3600 seconds (FM) or 30 seconds (binary)?
 
 [[write_timeout]]
 [float]
-=== `write_timeout`
+== Write timeout
 Maximum permitted duration for writing a response.
-Defaults to 30 seconds.
+Defaults to `30s` (30 seconds). (text)
+
+|====
+| APM Server binary | `apm-server.write_timeout`
+| Fleet-managed     | Maximum duration for writing a response
+|====
 
 [[shutdown_timeout]]
 [float]
-=== `shutdown_timeout`
+=== Shutdown timeout
 Maximum duration in seconds before releasing resources when shutting down the server.
-Defaults to 5 seconds.
+Defaults to `30s` (30 seconds). (text)
+
+|====
+| APM Server binary | `apm-server.shutdown_timeout`
+| Fleet-managed     | Maximum duration before releasing resources when shutting down
+|====
+
+// REVIEWER: Default 30 seconds (FM) or 5 seconds (binary)?
```

Comment:
30s - that's the default for the binary too. Looks like apm-server.yml is out of date, I'll fix that.